examples/IrisClassification/iris_classification.py

```diff
@@ -21,9 +21,9 @@ class IrisClassification(pl.LightningModule):
     def __init__(self, **kwargs):
         super(IrisClassification, self).__init__()
 
-        self.train_acc = Accuracy()
-        self.val_acc = Accuracy()
-        self.test_acc = Accuracy()
+        self.train_acc = Accuracy(task="multiclass", num_classes=3)
```

Comment:
Latest torchmetrics expects `task` and `num_classes` to be passed while using `Accuracy` - https://torchmetrics.readthedocs.io/en/stable/classification/accuracy.html